Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -30,13 +30,15 @@ WarpXParIter::WarpXParIter (ContainerType& pc, int level)
 {
 }
 
+WarpXParIter::WarpXParIter (ContainerType& pc, int level, MFItInfo& info)
+    : ParIter(pc, level, info.SetDynamic(WarpX::do_dynamic_scheduling))
```

Comment:
Is the ParIter on line 34 from amrex class?

Reply:
yes, I will add the namespace.